webpack/assets/javascripts/react_app/components/statistics/ChartBox.js

```diff
@@ -109,4 +108,8 @@ ChartBox.propTypes = {
   tip: PropTypes.string,
 };
 
+ChartBox.defaultProps = {
+  type: 'donut',
```

Comment:
Why donut is the default type when we have also [bar chart](https://github.com/theforeman/foreman/pull/5718/files#diff-c613cc11b2f0c91f5121a347d19b7be7R88)?
Could we keep the type as required prop:
```javascript
 <ChartBox type='donut' ...>
```